File: app/src/main/java/com/jcs/where/api/response/gourmet/dish/DeliveryTime.java
package com.jcs.where.api.response.gourmet.dish;

import java.util.ArrayList;

/**
 * Created by Wangsw  2021/4/27 17:04.
 */
public class DeliveryTime {

    /**
     * 立即配送时间
     */
    public String delivery_time = "";

    /**
     * 其他配送时间
     */
    public ArrayList<String> other_times = new ArrayList<>();

}

File: app/src/main/java/com/jcs/where/api/response/gourmet/dish/DeliveryTime.kt
package com.jcs.where.api.response.gourmet.dish

import java.util.*

/**
 * Created by Wangsw  2021/4/27 17:04.
 */
class DeliveryTime {

    /**
     * 立即配送时间
     */
    var delivery_time = ""

    /**
     * 其他配送时间
     */
    var other_times = ArrayList<String>()
}

class DeliveryTimeRetouch {

    var nativeSelected = false
    var time = ""
}